src/linux/mod.rs

```diff
@@ -70,9 +70,9 @@ impl Sandbox for LinuxSandbox {
         }
 
         // Create and apply seccomp filter.
-        let mut seccomp = Filter::new();
-        if !self.allow_networking {
-            seccomp.deny_networking()?;
+        let mut seccomp = NetworkFilter::new()?;
+        if self.allow_networking {
+            seccomp.allow_networking();
         }
         seccomp.apply()?;
```

Comment:
Wouldn't this be simpler without the `.allow_networking()` function that negates the `apply()`?

```suggestion
        if !self.allow_networking {
            let seccomp = NetworkFilter::new()?;
            seccomp.apply()?;
        }
```

Reply:
Haven't really considered the API after making the filter changes, but yeah this is better.